saleor/graphql/product/types/attributes.py

```diff
@@ -65,6 +65,7 @@ class Attribute(CountableDjangoObjectType):
 
     name = graphene.String(description=AttributeDescriptions.NAME)
     slug = graphene.String(description=AttributeDescriptions.SLUG)
+    type = AttributeTypeEnum(description=AttributeDescriptions.TYPE)
```

Comment:
Can we add descriptions here on this occasion? 👮‍♂️

Reply:
Do you mean to move the type description here? All descriptions are in`AttributeDescriptions` (`/saleor/graphql/product/descriptions.py`).

Reply:
Yes now I see, never mind, ignore my comment ;)